Scripts/ubuntu.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+command -v docker &> /dev/null  || { echo >&2 "Install docker https://www.docker.com"; exit 1; }
+ACTION=$1
```

Comment:
```suggestion
action=$1
```

with lowercase variables shellcheck will validate typos. with uppercase vars it will assume they're env vars if you mis-spell uses